src/udev/net/link-config.c

```diff
@@ -686,7 +673,27 @@ static int link_apply_rtnl_settings(Link *link, sd_netlink **rtnl) {
         return 0;
 }
 
-static int link_generate_new_name(Link *link, bool enable_name_policy) {
+static bool enable_name_policy(void) {
+        static int cached = -1;
+        bool b;
+        int r;
+
+        if (cached >= 0)
+                return cached;
+
+        r = proc_cmdline_get_bool("net.ifnames", &b);
+        if (r < 0)
+                log_warning_errno(r, "Failed to parse net.ifnames kernel command line option, ignoring: %m");
```

Comment:
maybe suffix `net.ifnames` → `net.ifnames=`, after all it expects an argument, and that's how we usually try to denote that fact in log messages.